mbed_app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "target_overrides": {
+        "*": {
+            "platform.stdio-baud-rate": 1152000,
```

Comment:
I think our examples still use 9600 by default?

Reply:
Oh yes, flashback from 1992 back to the years I used to use a  modem. :-)